akka-docs-dev/rst/scala/stream-graphs.rst

```diff
@@ -152,6 +152,20 @@ For defining a ``Flow[T]`` we need to expose both an inlet and an outlet:
 
 .. includecode:: code/docs/stream/StreamPartialFlowGraphDocSpec.scala#flow-from-partial-flow-graph
 
+Combining Sources and Sinks with simplified API
+-----------------------------------------------
+
+There is simplified API you can use to combine sources and sinks with junctions like: ``Broadcast[T]``, ``Balance[T]``,
+``Merge[In]`` and ``Concat[A]``. With method ``combine`` you don't need to connect junction
```

Comment:
".. without the need for using the Graph DSL. The combine method takes care of constructing the necessary graph underneath."